common-modules/common-tests/src/main/resources/__files/hydra/login_request_response_android.json

```diff
@@ -48,6 +48,6 @@
     "metadata": {
     }
   },
-  "request_url": "https://35.222.67.4:9000/oauth2/auth?client_id=oauth-scim-client-id&scope=offline_access&response_type=code&state=56d674f2-11dc-4350-86c7-33f11111f50a&redirect_uri=http://localhost:8002/auth-server/login&code_challenge_method=S256&code_challenge=lojRxBcyRjUtlUN6vLQ_jSzdntnhf_xdV5sCmQWANOQ&mobilePlatform=ANDROID&source=MOBILE APPS",
+  "request_url": "https://35.222.67.4:9000/oauth2/auth?client_id=oauth-scim-client-id&scope=offline_access&response_type=code&state=56d674f2-11dc-4350-86c7-33f11111f50a&redirect_uri=http://localhost:8002/auth-server/login&code_challenge_method=S256&code_challenge=lojRxBcyRjUtlUN6vLQ_jSzdntnhf_xdV5sCmQWANOQ&mobilePlatform=ANDROID&source=MOBILE APPS&appName=App Name_BTCDEV001",
```

Comment:
This seems like it should not be in the repo - this is a developer-specific URL

Reply:
Fixed review comment. Update URI to localhost.